projects/step-core/src/lib/directives/test-dataid.directive.ts

```diff
@@ -0,0 +1,14 @@
+import { Directive, ElementRef, inject, Input } from '@angular/core';
+
+@Directive({
+  selector: '[testDataId]',
+})
+export class TestDataIdDirective {
+  @Input('testDataId') testDataId: string | undefined;
+
+  private _el = inject(ElementRef);
```

Comment:
nativeElement type not specified.
```suggestion
  private _el = inject<ElementRef<HTMLElement>>(ElementRef);
```